README.rst

```diff
@@ -49,7 +49,7 @@ Starting the platform
 Please follow the output for next instructions.
 
 A python CLI and API client is available in the `renga-python
-<https://github/com/SwissDataScienceCenter/renga-python>`_ package.
+<https://github.com/SwissDataScienceCenter/renga-python/tree/development>`_ package.
```

Comment:
I wouldn't add the `/tree/development` since it will be problematic to merge to master later.

Reply:
fine... I'm just foreseeing comments of the type "none of the code you describe in your tutorial is in renga-python"

Reply:
@jirikuncar I added a note regarding development branches at the top instead.